test/js_test.go

```diff
@@ -475,6 +475,12 @@ func TestJetStreamSubscribe(t *testing.T) {
 			t.Fatalf("Timeout waiting for messages")
 		}
 	}
+
+	// Prevent invalid durable names
+	sub, err = js.SubscribeSync("baz", nats.Durable("test.durable"))
```

Comment:
Since unless you have a bug, sub should not be returned, I would do this:
```
if _, err := js.SubscribeSync(..); err != nats.ErrInvalidDurableName {
 ...
```